waspc/cli/src/Wasp/Cli/Command/CreateNewProject/StarterTemplates/Templating.hs

```diff
@@ -43,8 +44,16 @@ replaceTemplatePlaceholdersInPackageJsonFile appName projectName projectDir =
     Just absPackageJsonFile -> replaceTemplatePlaceholdersInFileOnDisk appName projectName absPackageJsonFile
 
 replaceTemplatePlaceholdersInFileOnDisk :: NewProjectAppName -> NewProjectName -> Path' Abs (File f) -> IO ()
-replaceTemplatePlaceholdersInFileOnDisk appName projectName =
-  updateFileContentWith (replacePlaceholders waspTemplateReplacements)
+replaceTemplatePlaceholdersInFileOnDisk appName projectName file = do
+  waspConfigPackagePath <- getPackageInstallationPath WaspConfigPackage
+  let waspTemplateReplacements =
+        [ ("__waspConfigPath__", waspConfigPackagePath),
+          ("__waspAppName__", show appName),
+          ("__waspProjectName__", show projectName),
+          ("__waspVersion__", defaultWaspVersionBounds)
+        ]
+  -- TODO: We do this in all files, but not all files have all placeholders
```

Comment:
But this how it was before also, right? Yeah, that is fine then. Btw this is afunction that is called upon a file, logic that decided which files this will be called upon is not in this function, so this comment should be referencing some other peice of code I guess, that I don't see from here at the moment.